src/scss/components/utility/_utility.scss

```diff
@@ -30,9 +37,42 @@
         &:hover,
         &:focus {
           color: $vpue-color-off-white;
+          background-color: $su-color-bright-red;
         }
       }
     }
   }
 }
 
+// Utility button placement
```

Comment:
This looks the same as before? Not sure can tell the difference (between old and new or between home and interior page.)